spec/lib/cap_authors_poller_spec.rb

```diff
@@ -82,14 +81,12 @@
     context 'with an existing author' do
       before do
         expect(Author).to receive(:find_by_cap_profile_id).with(author.cap_profile_id).and_return(author)
-        expect(author).to receive(:'save!').and_call_original
+        expect(author).to receive(:'save!')
       end
```

Comment:
Looks like this needs to be mocked because the factory has already saved the author.  Perhaps another option is to have the factory `build` the author instead of `create` it?  Then the specs will allow the code to save authors, as before.

Reply:
We never tested anything about the author *after* the `save!`, and this is not the test of the `Author` model, so there was no need to hit the DB here.  I made a similar change below at L123.

Reply:
Sorry, perhaps I'm confused about the intent here.  This is the cap authors poller and it seems like these specs are checking that it saves or does not save an author for various scenarios.  Some of the scenarios assume the author already exists and others do not.  AFAIK, when the `save!` is mocked, without calling the original, it does nothing any more.  Is it fair to assume that the new constraints made this fail for some reason (like the cap_profile_id is not unique)?

Reply:
Yeah, the specs were more loose with faked Author objects than our actual code was.